### PR TITLE
POWR-3527-3 config of struc sync replacing fixed blocks

### PR DESCRIPTION
**Pull request by ccdatapdx**

### CODE REVIEW

File: web/sites/default/config/block.block.views_block__db_search_group_news_block_featured_blog.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+uuid: 3432d519-ce0e-42ed-9f07-64a4b3677ebf
+langcode: en
```
Comment (joshuami) [inline web/sites/default/config/block.block.views_block__db_search_group_news_block_featured_blog.yml:2]: @ccdatapdx @RichardDavies , this file should not be added back in. I think there may be an issue with the order of git operations.

File: composer.lock [hunk 2]
```diff
@@ -22140,7 +22140,6 @@
     "aliases": [],
     "minimum-stability": "dev",
     "stability-flags": {
-        "drupal/better_exposed_filters": 10,
```
Comment (joshuami) [inline composer.lock:22143]: Hmm. I'm not sure this lock file should have changed either. 🤔

**Review by joshuami — CHANGES_REQUESTED** [review]

File: composer.lock (hunk 2 above)
Comment (RichardDavies) [reply composer.lock:22143]: I think this one is okay. I noticed my translation branch has the same change after doing some `composer requires`. so this appears to be expected.

File: web/sites/default/config/block.block.views_block__db_search_group_news_block_featured_blog.yml (hunk 1 above)
Comment (ccdatapdx) [reply web/sites/default/config/block.block.views_block__db_search_group_news_block_featured_blog.yml]: removed this file

**Pull request closed without merging**